src/SerialDataCollector.cpp

```diff
@@ -72,6 +72,9 @@ namespace splash
 
     std::string SerialDataCollector::getFullFilename(const Dimensions mpiPos, std::string baseFilename) const
     {
+        if (baseFilename.find(".h5") != std::string::npos)
```

Comment:
Just kept the old check. Though it might be good as it is as this e.g. allows `foobar.h5.myProcess` if one likes it that way ;)